extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/internal/model/TradfriVersion.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.tradfri.internal.model;
+
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
You should not use `@NonNull` but `@NonNullByDefault` and `@Nullable`.

Reply:
Still missing

Reply:
To be honest, I'm not very familiar with those annotations. Used them the first time for now. Please be patient with me on that. Hopefully I did it right now. :-)